WooCommerce/WooCommerceUITests/Mocks/mappings/jetpack-blogs/wc/reviews/products_on_review.json

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
This is the API call to get product data on the Reviews screen, the difference is on the query parameter where it has to include the product Ids.